src/main/java/noppes/npcs/entity/EntityNPCInterface.java

```diff
@@ -139,7 +139,8 @@ public abstract class EntityNPCInterface extends EntityCreature implements IEnti
 
 	public boolean updateClient = false;
 	public boolean updateAI = false;
-
+	public boolean projectilesKeepTerrain = false;
+	
```

Comment:
Move this into the DataStats class instead, that's where all the rest of the projectile attributes are kept!